zh_cn/news/_posts/2014-09-10-confoo-cfp.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: news_post
+title: "ConFoo 2015 is looking for Ruby speakers"
+author: "ylarrivee"
+translator:
+date: 2014-09-10 06:00:00 +0000
+lang: en
```

Comment:
I have add the translator and the lang.
thanks for your patience.

On Tue, Jun 2, 2015 at 5:27 PM Alex Shi <notifications@github.com> wrote:

> In zh_cn/news/_posts/2014-09-10-confoo-cfp.md
> <https://github.com/ruby/www.ruby-lang.org/pull/1105#discussion_r31505586>
> :
>
> > @@ -0,0 +1,30 @@
> > +---
> > +layout: news_post
> > +title: "ConFoo 2015 is looking for Ruby speakers"
> > +author: "ylarrivee"
> > +translator:
> > +date: 2014-09-10 06:00:00 +0000
> > +lang: en
>
> Specify lang to zh_cn
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruby/www.ruby-lang.org/pull/1105/files#r31505586>.
>